config/clusters/pangeo-hubs/support.values.yaml

```diff
@@ -35,3 +35,8 @@ prometheus:
         - secretName: prometheus-tls
           hosts:
             - prometheus.gcp.pangeo.2i2c.cloud
+    resources:
+      requests:
+        memory: 16Gi
+      limits:
+        memory: 16Gi
```

Comment:
16 Gi is quite arbitrary, but its 4 times larger than 4 Gi which it had by default.